go.mod

```diff
@@ -5,47 +5,47 @@ go 1.13
 require (
 	github.com/fsnotify/fsnotify v1.4.7
 	github.com/go-ini/ini v1.46.0
+	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
```

Comment:
do you know why this gets pulled in?

Reply:
Not yet. `go mod tidy` doesn't help, it keeps creeping in.

Reply:
looks like this is fixed

Reply:
Yes, `go mod why` is a great tool to solve mysteries. Turned out it was a temp folder with my hacks that brought the dep.